doc/user_doc/vic_installation/set_up_ops_user.md

```diff
@@ -0,0 +1,72 @@
+
+For information about how to create vSphere roles, see [vSphere Permissions and User Management Tasks](https://pubs.vmware.com/vsphere-65/topic/com.vmware.vsphere.security.doc/GUID-5372F580-5C23-4E9C-8A4E-EF1B4DD9033E.html) in the vSphere documentation.  For information about how to assign permissions to objects in the vSphere Inventory, see [Add a Permission to an Inventory Object](https://pubs.vmware.com/vsphere-65/topic/com.vmware.vsphere.security.doc/GUID-A0F6D9C2-CE72-4FE5-BAFC-309CFC519EC8.html).
+
+**Procedure**
+
+1. In the vSphere Web Client, create a user group, for example `VIC Ops Users`, and add the appropriate user accounts to the user group.
+
+   The best practice when assigning roles in vSphere is to assign the roles to user groups and then to add users to those groups, rather than assigning roles to the users directly.
+
+2. Go to **Administration** > **Roles** and create one role for each type of inventory object that VCHs need to access.
+
+   vSphere Integrated Containers Engine does not currently create the required roles, so you must create them manually. [Required VCH Roles](#required_roles) below lists the roles that you must create and the permissions that each role requires.
+
+3. Go to **Networking**, create a network folder, and place any distributed virtual switches that the VCHs will use into that folder.
+
+   The parent object of any distributed virtual switches that the VCH uses must be set to `Read-Only`, with **Propagate to Children** enabled. By placing distributed virtual switches in a network folder, you avoid setting an entire datacenter to `Read-Only`.
+
+2. Go to **Hosts and Clusters** and create a resource pool in which to deploy VCHs.
+
+   By creating a resource pool for VCHs, you can set the correct permissons on just that resource pool rather than on an entire host or cluster.
+
+5. In each of the **Hosts and Clusters**, **Storage**, and **Networking** views, select inventory objects and assign the user group and the appropriate role to each one.
+
+   [Inventory Objects to Which to Assign Permissions](#inventory_objects) below lists the inventory objects that VCHs access, and the role that you must assign to each one.
+
+**What to do next**
+
+Use `vic-machine create` to deploy VCHs, specifying an appropriate user account in the `--ops-user` option. Specify the inventory objects on which you have set permissions in the various `vic-machine create` options. For an example of a `vic-machine` command with the `--ops-user` option, see the section *Specify Different User Accounts for VCH Deployment and Operation* [Advanced Examples of Deploying a VCH](vch_installer_examples.md#ops-user).
```

Comment:
Reworded slightly:

Use `vic-machine create --ops-user=<user_account>` to deploy VCHs that operate with restricted privileges. Ensure that the various vSphere inventory objects that you specify as arguments have the user group with the appropriate role.